ccloud/docs/ccloud-demos-overview.rst

```diff
@@ -0,0 +1,189 @@
+Executed with a single command, it is a quick way to create fully managed components in |ccloud|, which you can then use for learning and building other demos.
+Please do not use this in a production environment.
+The script uses the |ccloud| CLI to dynamically do the following in |ccloud|:
+
+-  Create a new environment
+-  Create a new service account
+-  Create a new Kafka cluster and associated credentials
+-  Enable |sr-ccloud| and associated credentials
+-  Create a new KSQL app and associated credentials
+-  Create ACLs with wildcard for the service account
+-  Generate a local configuration file with all above connection information, useful for other demos/automation
+
+.. figure:: images/cloud-stack.png
+   :alt: image
+
+
+Client Code Examples
+--------------------
+
+If you are looking for code examples of producers writing to and consumers reading from |ccloud|, or producers and consumers using Avro with |sr-long|, refer to the :devx-examples:`client examples|clients/cloud/README.md`.
+It provides client examples written in various programming languages.
+
+.. figure:: ../clients/images/clients-all.png
+   :alt: image
+
+Confluent Cloud CLI
+-------------------
+
+The :devx-examples:`Confluent Cloud CLI demo|ccloud/beginner-cloud/README.md` is a fully scripted demo that shows users how to interact with |ccloud| using the |ccloud| CLI.
+It steps through the following workflow:
+
+-  Create a new environment and specify it as the default
+-  Create a new Kafka cluster and specify it as the default
+-  Create a user key/secret pair and specify it as the default
+-  Produce and consume with |ccloud| CLI
+-  Create a service account key/secret pair
+-  Run a Java producer: before and after ACLs
+-  Run a Java producer: showcase a Prefix ACL
+-  Run Connect and kafka-connect-datagen connector with permissions
+-  Run a Java consumer: showcase a Wildcard ACL
+-  Delete the API key, service account, Kafka topics, Kafka cluster, environment, and the log files
+
+Cloud ETL
+---------
+
+The :ref:`cloud ETL demo <cloud-etl>` showcases a cloud ETL solution leveraging all fully-managed services on |ccloud|.
+Using |ccloud| CLI, the demo creates a source connector that reads data from an AWS Kinesis stream into |ccloud|, then a Confluent Cloud KSQL application processes that data, and then a sink connector writes the output data into cloud storage in the provider of your choice (one of GCP GCS, AWS S3, or Azure Blob).
+
+.. figure:: ../../cloud-etl/docs/images/topology.png
+   :alt: image
+
+Hybrid Cloud
+------------
+
+The :ref:`hybrid cloud demo <quickstart-demos-ccloud>` and playbook showcase a hybrid Kafka deployment: one cluster is a self-managed cluster running locally, the other is a |ccloud| cluster.
+Data streams into topics both a local cluster and a cluster in |ccloud|, and |crep| copies the on-prem data to |ccloud| so that stream processing can happen in the cloud.
```

Comment:
```suggestion
Data streams into topics, in both the local cluster and the |ccloud| cluster. |crep| copies the on-prem data to |ccloud| so that stream processing can happen in the cloud.
```